templates/detail.html

```diff
@@ -10,13 +10,46 @@
         <!-- jQuery is already downloaded and installed for you -->
         <!-- In order to use jQuery in your JavaScript, this file needs to be added BEFORE your custom JavaScript -->
         <script type="text/javascript" src="/static/js/jquery.min.js"></script>
+
+
     </head>
     <body>
         <!-- Your HTML should be here -->
+        <a href="/"> <h3>Return to main page</h3></a>
+        <div class="container">
+            <div class="name"></div>
+            <div class="img"></div>
+            <h3>Types</h3>
+            <ul>
+                <div class="types"></div>
+            </ul>
+            <h3>Description </h3>        
+            <div class="description"></div>
```

Comment:
in general, here and elsewhere in this file, it's considered good practice to prefix any classes that you're going to be using in your javascript with "js-" so "js-description." If you're also using this class for CSS then use a separate one, so this would become:

`<div class="js-description desc"></div>`

or something like that